niworkflows/interfaces/utils.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Utilities
+
+"""
+
+import nibabel as nb
+from nipype.utils.filemanip import fname_presuffix
+from nipype.interfaces.base import File, OutputMultiPath, BaseInterfaceInputSpec, TraitedSpec
+from .base import SimpleInterface
+
+class CopyHeaderInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='the file we get the data from')
+    hdr_file = File(exists=True, mandatory=True, desc='the file we get the header from')
+
+class CopyHeaderOutputSpec(TraitedSpec):
+    out_file = OutputMultiPath(File(exists=True, desc='written file path'))
```

Comment:
I think this should just be a `File`, not an `OutputMultiPath`, unless I'm missing something?